src/AppBundle/Repository/PostRepository.php

```diff
@@ -28,6 +28,14 @@
 class PostRepository extends EntityRepository
 {
     /**
+     * @return Post[]
+     */
+    public function findAll()
```

Comment:
@bocharsky-bw,  for now we use the `findAll` only once in the admin controller.
https://github.com/symfony/symfony-demo/search?utf8=%E2%9C%93&q=findAll